test/molecule-role/docker-compose.yml

```diff
@@ -10,11 +10,21 @@ services:
       ADVERTISED_HOST: kafka
       ADVERTISED_PORT: 9092
   receiver:
-    image: "508573134510.dkr.ecr.eu-west-1.amazonaws.com/stackstate-receiver:release-1.12.1-rc1"
+    image: "508573134510.dkr.ecr.eu-west-1.amazonaws.com/stackstate-receiver:master"
     command: "-J-Xms128M -J-Xmx1G -J-XX:+ExitOnOutOfMemoryError -DconsoleLogging=true"
     ports:
       - 7077:7077
     depends_on:
       - kafka
     environment:
       KAFKA_BROKERS: kafka:9092
+  topic-api:
+    image: "508573134510.dkr.ecr.eu-west-1.amazonaws.com/stackstate-topic-api:master"
+    command: "-J-Xms128M -J-Xmx1G -J-XX:+ExitOnOutOfMemoryError -DconsoleLogging=true"
+    ports:
+      - 7070:7070
+    depends_on:
+      - kafka
+    environment:
+      KAFKA_BROKERS: kafka:9092
+      ZOOKEEPER_QUORUM: kafka
```

Comment:
why is this needed to the topic-api but not for the receiver ?

Reply:
The topic api uses zookeeper to learn about what's going on in kafka (what topics are there). The receiver only produces its own stuff so does not need this global overview